config/srasConfig.json

```diff
@@ -1,15 +1,11 @@
 {
-    "ignore": [
```

Comment:
Removing the ignore list will result in extra processing being done on these file types. The ignore check is the first check that is done which then skips everything else for these file types.

Reply:
I did some thinking about this, and I _do_ think we need the ignore list. Imagine a case where we:
1) want to apply some size check against PNG files
2) for _some_ PNG files (by name) we don't want to run any checks

This seems reasonable, but we'll need better wildcard support:
```json
{
  "ignore": [
    "*/bg/*.png"
  ]
}
```

or something like that.

However, it's also worth noting that I've only made the ignore list _optional_, not removed it entirely. However, removing it here is probably a bad idea since it'll remove that particular syntax as an example (although this should also live in the README. But that's another PR :D ). I'll fix that.